pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -989,6 +989,46 @@ func TestPanicThresholdPercentage(t *testing.T) {
 	}
 }
 
+func TestMinNonZeroReplicasAnnotation(t *testing.T) {
+	cases := []struct {
+		name      string
+		pa        *PodAutoscaler
```

Comment:
nit: don't need to setup each test with a PA if you're just tweaking annotations

```suggestion
		annotations        map[string]string
```